io.github.JakubMelka.Pdf4qt.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "io.github.JakubMelka.Pdf4qt",
+    "runtime": "org.kde.Sdk",
+    "runtime-version": "6.5",
+    "sdk": "org.kde.Sdk",
+    "command": "Pdf4QtViewerProfi",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=ipc"
+    ],
+    "cleanup": [
+        "/pdf4qt",
+        "/vcpkg"
+    ],
+    "modules": [ 
+        {
+            "name": "pdf4qt",
+            "buildsystem": "cmake-ninja",
+            "post-install": [
+                "echo 'Finishing BUILD - move plugins into right place'",
+                "./postbuild.sh"
```

Comment:
these command should be inlined. and the `echo` is totally unecessary.

Reply:
Inlined, echo removed.